src/core/include/antioch/chemical_mixture.h

```diff
@@ -569,6 +585,24 @@ namespace Antioch
     return;
   }
 
+  template<typename CoeffType>
+  inline
+  const ChemicalMixture<CoeffType> & ChemicalMixture<CoeffType>::chemical_mixture() const
+  {
+     antioch_deprecated();
+     return *this;
+  }
+
+  template<typename CoeffType>
+  template<typename ThermoEvaluator>
+  inline
+  const ThermoEvaluator & ChemicalMixture<CoeffType>::thermo_evaluator() const
+  {
+     std::cerr << "What the hell are you doing here? This backward compatibility place should not be explored" << std::endl;
```

Comment:
Well that's a place that exists strictly so the compiler can compile. I was not sure what to do here, the only thing I'm convinced is that this piece of code should be short-lived.